contents/handbook/people/offboarding.md

```diff
@@ -8,40 +8,51 @@ Offboarding team members can be a sensitive time. The aim of this policy is to c
 
 Very infrequently, we may have long term contractors working for PostHog, acting essentially like a permanent employee. In this case, the process below is exactly the same.  This offboarding policy *does not* apply to regular contractors who are doing short term work for us.
 
-## Communicating Departures
+## Voluntary Departure
 
-PostHog cannot always provide context around why people are leaving when they do.
+In this case, the team member chooses to leave PostHog. 
 
-In the case of voluntary departure, we will ask the team member if they wish to share what they're up to next with the team.
+We ask for 30 days of notice by default (unless locally a different maximum or minimum limit applies), and for you to work during that notice period. This is so we have some time to find someone to hire and to enable a handover.
 
-In the case of involuntary departure, we are unlikely to communicate any reasons to the rest of the company other than the fact the person has left. Whilst transparency is something we value greatly, this information affects individuals' privacy. 
```

Comment:
I would like to open a discussion around this. This might lead to some uncertainties in the team. We should be able to find a middle ground here and at least emphasis that we have given the team member the chance to act on received feedback.

Reply:
How would you propose we re-phrase this? Happy to go with a recommendation here - I don't think the way it's written is that far off target at the moment.

Reply:
I think it would badly affect team morale is someone was let go _seemingly_ with no reason. For instance if Sean was just let go and we were told only to forget he ever worked here, that'd lead to distrust and rumors. Fortunately the situation was handled much more transparently by James H. and we avoided that outcome. IMO the degree of openness should be similar in the future.

Reply:
> How would you propose we re-phrase this? Happy to go with a recommendation here - I don't think the way it's written is that far off target at the moment.

"In the case of involuntary departure, we will aim to be as transparent as possible about the reasons behind the departure, while respecting the individuals' privacy."

Reply:
Yep looks good to me - should just be _individual's_.